Source/Core/Core/PowerPC/Jit64/Jit.h

```diff
@@ -98,6 +98,12 @@ class Jit64 : public JitBase, public QuantizedMemoryRoutines
   void WriteExternalExceptionExit();
   void WriteRfiExitDestInRSCRATCH();
   void WriteIdleExit(u32 destination);
+  void WriteBranchWatch(u32 origin, u32 destination, UGeckoInstruction inst,
+                        BitSet32 registers_in_use);
+  void WriteBranchWatchDestInRSCRATCH(u32 origin, UGeckoInstruction inst);
```

Comment:
While the fact that this function doesn't have a `registers_in_use` parameter does provide a hint, I think it's not obvious that this function doesn't do any saving of registers. I assume you left out the `register_in_use` parameter because no called needed to save any registers other than RSCRATCH? Personally I think the intent of the code would become clearer at the call site if the function has a `registers_in_use` parameter that the caller populates with just RSCRATCH.

Reply:
I agree. Thanks.